spec/models/api/hses_spec.rb

```diff
@@ -21,28 +21,86 @@
   end
 
   describe "Issues" do
-    let(:issues) { Api::Hses::Issues.new(user: user) }
+    context "with params argument" do
+      let(:issues) {
+        Api::Hses::Issues.new(
+          user: user,
+          params: ActionController::Parameters.new({page: "2"})
+        )
+      }
 
-    describe "#initialize" do
-      it "sets @username" do
-        expect(issues.instance_variable_get(:@username)).to eq user["uid"]
+      describe "#initialize" do
+        it "sets @username" do
+          expect(issues.instance_variable_get(:@username)).to eq user["uid"]
```

Comment:
[suggestion] mmm, if it's not publicly available, we shouldn't be reaching in to grab it. i don't think we need these tests!